### ISSUE TITLE
Update models for multiclass

### ISSUE THREAD

**Issue by daniprec** [UCA-Datalab/better_nilm#16]
Update our pytorch models to perform multiclass classification.
See [cross entropy loss](https://pytorch.org/docs/stable/generated/torch.nn.CrossEntropyLoss.html#torch.nn.CrossEntropyLoss).

**Comment by daniprec** [comment]
:warning: **Notice:** This project is no longer actively maintained, and this issue won't be addressed. Feel free to continue the discussion, but be aware that there won't be further updates.
